netstandard/ref/System.Security.Cryptography.cs

```diff
@@ -626,10 +710,11 @@ public abstract partial class MD5 : System.Security.Cryptography.HashAlgorithm
         public static new System.Security.Cryptography.MD5 Create() { throw null; }
         public static new System.Security.Cryptography.MD5 Create(string algName) { throw null; }
     }
+    [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
     public sealed partial class MD5CryptoServiceProvider : System.Security.Cryptography.MD5
     {
         public MD5CryptoServiceProvider() { }
-        protected override void Dispose(bool disposing) { }
+        protected sealed override void Dispose(bool disposing) { }
```

Comment:
@weshaggard Does saying `sealed override` here compel all implementations to seal it?  I'm not sure why it's `sealed override` in core, since it's a sealed type.  (Applies to all of the `sealed override`s)

Reply:
I don't feel it's necessary, no.  Unless there's something special about sealed overrides on sealed types that I don't understand.

I'd, personally, replace every `sealed override` with `override`, checking through the file they all appear in sealed types.

Reply:
Addressed by 9ed3efc